algorithms/common/buffer/priortized_replay_buffer.py

```diff
@@ -26,16 +26,24 @@ class PrioritizedReplayBuffer(ReplayBuffer):
     https://github.com/openai/baselines/blob/master/baselines/deepq/replay_buffer.py
```

Comment:
need to modify this comment.
We changed a lot of the implementation. (not taken)